data/strings/strings.txt

```diff
@@ -21140,46 +21140,46 @@
 
   [list_settings]
     tags = android,ios
-    en = List Settings
```

Comment:
There is already a well-translated [edit] key in strings.txt, there is no need to change [list_settings]. It can be removed, if we proceed with changing `List Settings` to `Edit`. But again, thinking about the future when list will be shared with other users, will `Edit` be better than `List Settings`?

Reply:
I think ``` Edit ``` is a much more descriptive name than ``` List settings ``` and I don't see any reason why it wouldn't fit well with the share option. I will be removing List settings entirely then.